src/ng/jsonpCallbacks.js

```diff
@@ -11,9 +11,9 @@
  */
 var $jsonpCallbacksProvider = function() {
   this.$get = ['$window', function($window) {
-    var counter = 0;
-    $window.angular.callbacks = {};
     var callbackMap = {};
+    var callbacks = $window.angular.callbacks = $window.angular.callbacks || {};
```

Comment:
Nope :smiley:

This is basically doing:
```js
foo = foo || {};   // foo <--- $window.angular.callbacks
var bar = foo;   // bar  <--- callbacks
```

The idea is that if `$window.angular.callbacks` has already been initialized (e.g. by another app running on the same page or by a different instance of `$jsonpCallbacks` created by a different injector), it should not be re-initialized/overwritten.